tests-pytest/action.yml

```diff
@@ -1,41 +1,70 @@
-name: "Run a test suite using the pytest framework."
-description: "Run a test suite using the pytest framework."
+name: >
+  Tests pytest action
+
+description: >
+  Run a test suite using `the pytest framework
```

Comment:
I saw one make a PR to "fix" the inclusion of a "the" in a link. I'd suggest removing outside the link text.

Reply:
Just reformated this sented as:

```
Run a test suite using `pytest <https://docs.pytest.org/en/stable/>`_.
```